CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # Changelog
 
-## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/1.27.0...HEAD)
+## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/1.28.0...HEAD)
+
+
+## [1.28.0](https://github.com/OpenLineage/OpenLineage/compare/1.27.0...1.28.0)
 
 ### Added
 
 * **Java: enable specifying custom SSL context** [`#3444`](https://github.com/OpenLineage/OpenLineage/pull/3444) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)
   *Enable providing configuration for SSL context within HTTP transport.
 * **Spark: make Spark nodes filtering configurable.** [`#3442`](https://github.com/OpenLineage/OpenLineage/pull/3442) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)
   *Spark integration filters OpenLineage events for specific plan node classes. This can be now extended with extra config entries: `allowedSparkNodes` and `deniedSparkNodes`. See [Spark Configuration documentation](https://openlineage.io/docs/integrations/spark/configuration/spark_conf) for more details. *
```

Comment:
Not sure but probably this fixes formatting for this:
<img width="1011" alt="image" src="https://github.com/user-attachments/assets/94082a77-0654-4fc2-a963-260a070fdaab" />

```suggestion
  *Spark integration filters OpenLineage events for specific plan node classes. This can be now extended with extra config entries: `allowedSparkNodes` and `deniedSparkNodes`. See [Spark Configuration documentation](https://openlineage.io/docs/integrations/spark/configuration/spark_conf) for more details.*
```

Reply:
Fixed.